patches/net/minecraft/client/multiplayer/ClientCommonPacketListenerImpl.java.patch

```diff
@@ -35,6 +35,16 @@
 +            return;
 +        }
 +
++        if (p_295727_.payload() instanceof net.neoforged.neoforge.network.payload.CommonVersionPayload) {
++            // no-op
```

Comment:
Explain in this comment why the `c:version` payload is a no-op. (Alternatively, explain it in `CommonVersionPayload` and reference that from this comment.)